macos/mac_os_resource_snd.ksy

```diff
@@ -0,0 +1,280 @@
+meta:
+  id: mac_os_resource_snd
+  title: Classic MacOS Sound Resource
+  application: Sound Manager
+  endian: be
+  license: MIT
+  xref:
+    wikidata: Q7564684
+    mac-os-resource-type: 'snd '
+doc-ref: "https://developer.apple.com/library/archive/documentation/mac/pdf/Sound/Sound_Manager.pdf"
+doc: |
+  Sound resources were introduced in Classic MacOS with the Sound Manager program.
+  They can contain sound commands to generate sounds with given frequencies as well as sampled sound data.
+  They are mostly found in resource forks, but can occasionally appear standalone or embedded in other files.
+seq:
+  - id: format
+    type: u2
+  - id: num_data_formats
+    -orig-id: number_of_data_formats
+    type: u2
+    if: format == 1
+  - id: data_formats
+    type: data_format
+    repeat: expr
+    repeat-expr: num_data_formats
+    if: format == 1
+  - id: reference_count
+    type: u2
+    if: format == 2
+  - id: num_sound_commands
+    -orig-id: number_of_sound_commands
+    type: u2
+  - id: sound_commands
+    type: sound_command
+    repeat: expr
+    repeat-expr: num_sound_commands
+types:
+  data_format:
+    seq:
+      - id: id
+        type: u2
+        enum: data_type
+      - id: options
+        type: u4
```

Comment:
It's strange that in the rest of the spec you've described more or less obvious things like `num_channels` &ndash; _number of channels in sample_ (which is not wrong, just a bit redundant), while this broad name `options` doesn't have any description. I think that this field is the one that would deserve an explanation the most.

Reply:
The options are passed to the SndNewChannel as the "init" parameter.

There are some documented values for this field (Page 2-91 in https://developer.apple.com/library/archive/documentation/mac/pdf/Sound/Sound_Manager.pdf), but I don't know if it is feasible to extract them as own instances.

What do you think?

Reply:
I have added some instances and enums to better describe the contents of the options field